src/node.cc

```diff
@@ -3637,6 +3637,8 @@ static void ParseArgs(int* argc,
   const char** new_v8_argv = new const char*[nargs];
   const char** new_argv = new const char*[nargs];
   const char** local_preload_modules = new const char*[nargs];
+  bool use_bundled_ca = false;
+  bool use_openssl_ca = false;
```

Comment:
using a simple bit field might work also. not critical tho... but if you did...

```c++
int use_ca_flag = 0;
// ...
use_ca_flag |= 1;
use_ca_flag |= 2;
//
if (use_ca_flag == 3) {
  // ... error
}
```
You could get by with only a single variable and a single condition check when reporting the error.

It's extremely minor so feel free to ignore tho.